### DISCUSSION TITLE
Instant Reveal

### DISCUSSION THREAD

**Discussion by giacomobandinelli** [hashlips-lab/nft-erc721-collection#379] [General]
Hello, I would like to know what is the easiest way to remove the reveal function.
I would like to do so because I would like to have an instant reveal of the NFTs. 
Thank you in advance for your support.

**Comment by DevCores** [comment]
> Hello, I would like to know what is the easiest way to remove the reveal function. I would like to do so because I would like to have an instant reveal of the NFTs. Thank you in advance for your support.

```
string public uriPrefix = 'ipfs://QmXRxpgJRgALD3WR3yktryAsDwfTf9yySoov2NEEMd1JP8/';
bool public revealed = true;
```
This will solve your problem
